examples/settings.py

```diff
@@ -17,11 +17,15 @@
     "color_sensor": True,  # RGB sensor (default: ON)
     "semantic_sensor": False,  # semantic sensor (default: OFF)
     "depth_sensor": False,  # depth sensor (default: OFF)
-    "ortho_sensor": False,  # Orthographic RGB sensor (default: OFF)
+    "ortho_rgb_sensor": False,  # Orthographic RGB sensor (default: OFF)
```

Comment:
@bigbike This should probably be renamed to rgba. I wasn't aware they returned RGBA info until now.